src/System.Drawing.Primitives/src/System/Drawing/Point.cs

```diff
@@ -0,0 +1,280 @@
+        ///       Translates a <see cref='System.Drawing.Point'/> by the negative of a given <see cref='System.Drawing.Size'/> .
+        ///    </para>
+        /// </devdoc>        
+        public static Point Subtract(Point pt, Size sz)
+        {
+            return new Point(pt.X - sz.Width, pt.Y - sz.Height);
+        }
+
+        /// <devdoc>
+        ///   Converts a PointF to a Point by performing a ceiling operation on
+        ///   all the coordinates.
+        /// </devdoc>
+        public static Point Ceiling(PointF value)
+        {
+            return new Point((int)Math.Ceiling(value.X), (int)Math.Ceiling(value.Y));
+        }
+
+        /// <devdoc>
+        ///   Converts a PointF to a Point by performing a truncate operation on
+        ///   all the coordinates.
+        /// </devdoc>
+        public static Point Truncate(PointF value)
+        {
+            return new Point((int)value.X, (int)value.Y);
+        }
+
+        /// <devdoc>
+        ///   Converts a PointF to a Point by performing a round operation on
+        ///   all the coordinates.
+        /// </devdoc>
+        public static Point Round(PointF value)
+        {
+            return new Point((int)Math.Round(value.X), (int)Math.Round(value.Y));
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Specifies whether this <see cref='System.Drawing.Point'/> contains
+        ///       the same coordinates as the specified <see cref='System.Object'/>.
+        ///    </para>
+        /// </devdoc>
+        public override bool Equals(object obj)
+        {
+            if (!(obj is Point)) return false;
+            Point comp = (Point)obj;
+            // Note value types can't have derived classes, so we don't need 
+            // to check the types of the objects here.
+            return comp.X == X && comp.Y == Y;
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Returns a hash code.
+        ///    </para>
+        /// </devdoc>
+        public override int GetHashCode()
+        {
+            return unchecked(_x ^ _y);
+        }
+
+        /// <devdoc>
+        ///    Translates this <see cref='System.Drawing.Point'/> by the specified amount.
+        /// </devdoc>
+        public void Offset(int dx, int dy)
+        {
+            X += dx;
+            Y += dy;
+        }
+
+        /// <devdoc>
+        ///    Translates this <see cref='System.Drawing.Point'/> by the specified amount.
+        /// </devdoc>
+        public void Offset(Point p)
+        {
+            Offset(p.X, p.Y);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Converts this <see cref='System.Drawing.Point'/>
+        ///       to a human readable
+        ///       string.
+        ///    </para>
+        /// </devdoc>
+        public override string ToString()
+        {
+            return "{X=" + X.ToString(CultureInfo.CurrentCulture) + ",Y=" + Y.ToString(CultureInfo.CurrentCulture) + "}";
```

Comment:
Isn't the default for ToString() to use the current culture?  If so, this could just be:
```
return "{X=" + X.ToString() + ",Y=" + Y.ToString() + "}";
```
While it could be simplified further to avoid the ToString() calls at all, explicitly adding these ToStrings does have a minor benefit on minimizing allocations.